leak.c

```diff
@@ -0,0 +1,29 @@
+#include <stdio.h>
```

Comment:
This test program should live in the `test/` directory, along with it's respective Dockerfile and the rest of the project's tests.